shuup/core/migrations/0005_data_migrate_language_fallback.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.apps import apps as django_apps
+from django.conf import settings
+from django.db import migrations
+from django.utils import translation
+from parler.models import TranslationDoesNotExist
+
+
+def to_language_codes(languages):
+    languages = languages or (translation.get_language(),)
+    if languages and isinstance(languages[0], (list, tuple)):
+        # `languages` looks like a `settings.LANGUAGES`, so fix it
+        languages = [code for (code, name) in languages]
+    return languages
+
+
+def save_values_from_default_to_lang(default_translation, translation_object, fields):
+
+    for field in fields:
+        default_value = getattr(default_translation, field.name, '')
+        orig_value = getattr(translation_object, field.name)
+
+        if not orig_value and default_value != orig_value:
+            setattr(translation_object, field.name, default_value)
+
+        translation_object.save()
+
+
+def add_missing_values(model, default_lang, languages):
+
+    objects = model.objects.all()
+
+    translation_model = model.translations.related.related_model
+    base_fields = translation_model.get_translated_fields()
+
+    translation_model_fields = [
+        f for f in translation_model._meta.fields
+        if f.name in base_fields and not f.unique
+    ]
+
+    for obj in objects:
+        # model being translatable does not guarantee translations exist
+        if obj.translations.count():
+            default_translation = obj.get_translation(default_lang)
```

Comment:
This will fail if the object doesn't have translation in PARLER_DEFAULT_LANGUAGE_CODE. That happens some times since I think only currently active language is required most of the `MultiLanguageModelForm`.